### ISSUE TITLE
IA-generate icon

### ISSUE THREAD

**Issue by MontseTelefonica** [Telefonica/mistica-design#1992]
### Contribution guidelines

- [X] Have you read the [Contributing Guidelines on Mística](https://brandfactory.telefonica.com/d/iSp7b1DkYygv/n-a#/contribute/how-to-contribute)?
- [X] Have you read the [Contributing document for icons](https://github.com/Telefonica/mistica-icons/blob/production/CONTRIBUTING.md)?

### What brand do you need to include these icons?

Movistar, Vivo, O2, Telefónica, Blau

### Is your icon request related to a need?

For the CMS we need an IA icon to display in those functionalities that use IA to work in the media manager. Right now we are usinga Muistica icon from the material library but to use it in tags we need it yo be Mistica icon. 
<img width="595" alt="image" src="https://github.com/user-attachments/assets/fc63e61c-5f4d-4ba5-8891-cb488524b6b7" />
<img width="204" alt="image" src="https://github.com/user-attachments/assets/6500fc10-09fd-48b0-a2ab-32778c2a8d1c" />

This is the idea of the icon we need

### Additional context?

https://www.figma.com/design/WnSulrbYF6ATfgVrkhg7KN/AIgenerated?node-id=0-1

### Attach your svg icon with 3 weights (light / regular / filled)

![AIGenerated-regular](https://github.com/user-attachments/assets/ab446709-9b82-4d8b-bdd7-e944b8cf41a5)
![AIGenerated-filled](https://github.com/user-attachments/assets/a27e46e5-c8c2-4a34-a920-1df6a2467db9)
![AIGenerated-light](https://github.com/user-attachments/assets/9fb58235-8917-4d89-ac1e-ec3d95acd39e)

**Comment by yceballost** [comment]
@MontseTelefonica is the icon mirrored?

**Comment by MontseTelefonica** [comment]
Yes, I changed a bit so it's not identical to the material one. I also moved the bottom star 2px to the right. But feel free to make the changes that Mistica thinks are better.
